OWNERS

```diff
@@ -2,7 +2,6 @@
 
 approvers:
 - amwat # oncall
-- BenTheElder # oncall
```

Comment:
`# - BenTheEnderl # emeritus`?

Reply:
https://github.com/kubernetes/test-infra/issues/10747 ?

Reply:
all I care about here is giving you some recognition as emeritus, this is the laziest way for now (we do this for leads in sigs.yaml via another field)

Reply:
got it, my brain missed the `#`, this is done.